diff --git a/modules/contrib/filter_script_tags/src/Plugin/Filter/FilterScriptTags.php b/modules/contrib/filter_script_tags/src/Plugin/Filter/FilterScriptTags.php
index 10b94ae..2ab52d8 100644
--- a/modules/contrib/filter_script_tags/src/Plugin/Filter/FilterScriptTags.php
+++ b/modules/contrib/filter_script_tags/src/Plugin/Filter/FilterScriptTags.php
@@ -122,9 +122,10 @@ private function scriptFilterScript(string $string, array $whitelist_regex = [])
 
         // Set flag to indicate whether we have modified $html_input and therefore need to return it.
         $send_new_html = false;
+        $options = LIBXML_HTML_NOIMPLIED;
 
         $dom = new \DOMDocument();
-        $dom->loadHTML($html_input);
+        $dom->loadHTML($html_input ? $options : false);
         $body = $dom->getElementsByTagName('body');
         $body = $body[0];
 
