diff --git a/modules/contrib/default_content/src/Importer.php b/modules/contrib/default_content/src/Importer.php
index 7a4f781..06b63b0 100644
--- a/modules/contrib/default_content/src/Importer.php
+++ b/modules/contrib/default_content/src/Importer.php
@@ -12,6 +12,7 @@
 use Drupal\user\EntityOwnerInterface;
 use Symfony\Component\EventDispatcher\EventDispatcherInterface;
 use Symfony\Component\Serializer\Serializer;
+use Drupal\Core\Entity\EntityTypeInterface;
 
 /**
  * A service for handling import of default content.
@@ -128,11 +129,11 @@ public function importContent($module) {
         // if ($reflection->implementsInterface(ConfigEntityInterface::class)) {
         //   continue;
         // }
-        $entityType = new \Drupal\Core\Entity\EntityType($entity_type->getGroup());
+        $entity_type_id = $entity_type->getGroup();
          // We are only interested in importing content entities.
-        if ($entityType->implementsInterface(EntityTypeInterface::getGroup())) {
-           continue;
-        }
+        // if ($entityType->implementsInterface(EntityTypeInterface::class)) {
+        //    continue;
+        // }
         if (!file_exists($folder . '/' . $entity_type_id)) {
           continue;
         }
