diff --git a/modules/contrib/flood_unblock/flood_unblock.info.yml b/modules/contrib/flood_unblock/flood_unblock.info.yml
index 43d8bc5..841379f 100644
--- a/modules/contrib/flood_unblock/flood_unblock.info.yml
+++ b/modules/contrib/flood_unblock/flood_unblock.info.yml
@@ -1,6 +1,6 @@
 name: Flood Unblock
 type: module
-description: "Interface for unblocking ip's that are blocked by the flood table."
+description: "Interface for unblocking the IP's that are blocked by the flood table."
 package: user
 configure: flood_unblock.form
 # version: VERSION
diff --git a/modules/contrib/flood_unblock/flood_unblock.links.menu.yml b/modules/contrib/flood_unblock/flood_unblock.links.menu.yml
index 267bf51..6fbf879 100644
--- a/modules/contrib/flood_unblock/flood_unblock.links.menu.yml
+++ b/modules/contrib/flood_unblock/flood_unblock.links.menu.yml
@@ -1,6 +1,6 @@
 flood_unblock.admin:
-  title: 'Flood unblock'
-  description: 'List all user blocked by the flood table.'
+  title: 'Flood Unblock'
+  description: 'List all users that are blocked by the flood table.'
   parent: entity.user.collection
   route_name: flood_unblock.form
   menu_name: devel
diff --git a/modules/contrib/flood_unblock/flood_unblock.links.task.yml b/modules/contrib/flood_unblock/flood_unblock.links.task.yml
index 6dc3ec1..a285c52 100644
--- a/modules/contrib/flood_unblock/flood_unblock.links.task.yml
+++ b/modules/contrib/flood_unblock/flood_unblock.links.task.yml
@@ -1,5 +1,5 @@
 flood_unblock.flood_unblock_task:
-  title: 'Flood unblock'
+  title: 'Flood Unblock'
   route_name: flood_unblock.form
   base_route: entity.user.collection
   weight: 40
\ No newline at end of file
diff --git a/modules/contrib/flood_unblock/flood_unblock.permissions.yml b/modules/contrib/flood_unblock/flood_unblock.permissions.yml
index 19b5a58..8b91016 100644
--- a/modules/contrib/flood_unblock/flood_unblock.permissions.yml
+++ b/modules/contrib/flood_unblock/flood_unblock.permissions.yml
@@ -1,5 +1,5 @@
 # In my_module.permissions.yml file.
 access flood unblock:
   title: 'Access the flood unblock module'
-  description: 'Allow user to flood unblock.'
+  description: 'Allow user to unblock ip-addresses from flood table.'
   restrict access: TRUE
diff --git a/modules/contrib/flood_unblock/help_topics/flood_unblock.overview.html.twig b/modules/contrib/flood_unblock/help_topics/flood_unblock.overview.html.twig
index bfdd787..59d5a23 100644
--- a/modules/contrib/flood_unblock/help_topics/flood_unblock.overview.html.twig
+++ b/modules/contrib/flood_unblock/help_topics/flood_unblock.overview.html.twig
@@ -9,12 +9,12 @@ related:
 {% set perms_url = render_var(url('user.admin_permissions')) %}
 <h2>{% trans %}Goal{% endtrans %}</h2>
 <p>{% trans %}Flood Unblock provides an <a href="{{ fu_url }}">interface</a> that
-  makes possible for site administrators to remove IP adresses and user ID's from the
+  makes possible for site administrators to remove IP addresses and user ID's from the
   flood table.{% endtrans %}</p>
 <p>{% trans %}Drupal prevents brute force attacks on accounts. It does so by
     refusing login attempts when more than 5 attempts failed. The amount
     of failed logins is recorded in the table 'flood'. You can either
-    wait before trying to login again or clean the flood table with the
+    wait before trying to login again or clear the flood table with the
     procedure below.{% endtrans %}</p>
 <h3>{% trans %}Permissions{% endtrans %}</h3>
 <p>{% trans %}Users with the <em>access flood unblock</em> <a href="{{ perms_url }}">permission</a> can access the <a href="{{ fu_url}}"><em>Flood Unblock</em></a> page.{% endtrans %}</p>
diff --git a/modules/contrib/flood_unblock/src/FloodUnblockManager.php b/modules/contrib/flood_unblock/src/FloodUnblockManager.php
index 9a839aa..c9d73cf 100644
--- a/modules/contrib/flood_unblock/src/FloodUnblockManager.php
+++ b/modules/contrib/flood_unblock/src/FloodUnblockManager.php
@@ -104,7 +104,7 @@ public function getEntries() {
             $user_link = $user->toLink($user->getAccountName());
           }
           else {
-            $user_link = $this->t('Deleted user: @user', ['@user' => $result->uid]);
+            $user_link = $this->t('Deleted User: @user', ['@user' => $result->uid]);
           }
           $entries[$result->identifier]['username'] = $user_link ?? NULL;
         }
diff --git a/modules/contrib/flood_unblock/src/Form/FloodUnblockAdminForm.php b/modules/contrib/flood_unblock/src/Form/FloodUnblockAdminForm.php
index 422ebf3..d95452c 100644
--- a/modules/contrib/flood_unblock/src/Form/FloodUnblockAdminForm.php
+++ b/modules/contrib/flood_unblock/src/Form/FloodUnblockAdminForm.php
@@ -76,7 +76,7 @@ public function buildForm(array $form, FormStateInterface $form_state) {
 
     $header = [
       'blocked' => $this->t('Blocked'),
-      'type' => $this->t('Type of block'),
+      'type' => $this->t('Block Type'),
       'count' => $this->t('Count'),
       'uid' => $this->t('Account name'),
       'ip' => $this->t('IP Address'),
@@ -95,19 +95,19 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     }
 
     $form['top_markup'] = [
-      '#markup' => $this->t('<p>Use the table below to view the available flood entries. You can clear seperate items.</p>'),
+      '#markup' => $this->t('<p>Use the table below to view the available flood entries. You can clear separate items.</p>'),
     ];
 
     $form['table'] = [
       '#type' => 'tableselect',
       '#header' => $header,
       '#options' => $options,
-      '#empty' => $this->t('There are no failed logins at this time.'),
+      '#empty' => $this->t('There are no failed users logins at this time.'),
     ];
 
     $form['submit'] = [
       '#type' => 'submit',
-      '#value' => $this->t('Clear flood'),
+      '#value' => $this->t('Remove flood'),
     ];
 
     if (count($entries) == 0) {
@@ -128,7 +128,7 @@ public function validateForm(array &$form, FormStateInterface $form_state) {
       return $selected !== 0;
     });
     if (empty($selected_entries)) {
-      $form_state->setErrorByName('table', $this->t('Please make a selection.'));
+      $form_state->setErrorByName('table', $this->t('Please make a selection entries.'));
     }
   }
 
diff --git a/modules/contrib/flood_unblock/tests/src/Functional/FloodUnblockUiPageTest.php b/modules/contrib/flood_unblock/tests/src/Functional/FloodUnblockUiPageTest.php
index d472276..9a8e27b 100644
--- a/modules/contrib/flood_unblock/tests/src/Functional/FloodUnblockUiPageTest.php
+++ b/modules/contrib/flood_unblock/tests/src/Functional/FloodUnblockUiPageTest.php
@@ -50,7 +50,7 @@ public function testFloodUnblockUiPageAdminUser() {
 
     // Test that there is an empty flood list.
     $this->assertSession()
-      ->pageTextContains('There are no failed logins at this time.');
+      ->pageTextContains('There are no failed users logins at this time.');
   }
 
   public function testFloodUnblockUiPageSimpleUser() {
