diff --git a/modules/contrib/flood_unblock/src/FloodUnblockManager.php b/modules/contrib/flood_unblock/src/FloodUnblockManager.php
index 9a839aa..e73a2d9 100644
--- a/modules/contrib/flood_unblock/src/FloodUnblockManager.php
+++ b/modules/contrib/flood_unblock/src/FloodUnblockManager.php
@@ -9,7 +9,7 @@
 use Drupal\Core\Messenger\MessengerInterface;
 use Drupal\Core\StringTranslation\StringTranslationTrait;
 
-class FloodUnblockManager {
+class FloodUnblockManager implements FloodUnblockManagerInterface {
 
   use StringTranslationTrait;
 
@@ -57,10 +57,7 @@ public function __construct(Connection $database, FloodInterface $flood, ConfigF
   }
 
   /**
-   * Generate rows from the entries in the flood table.
-   *
-   * @return array
-   *   Entries of the flood table grouped by identifier (UID/IP).
+   * {@inheritdoc}
    */
   public function getEntries() {
     $entries = [];
@@ -144,7 +141,7 @@ public function getEntries() {
   }
 
   /**
-   * The function that clear the flood.
+   * {@inheritdoc}
    */
   public function flood_unblock_clear_event($event, $identifier) {
     $txn = $this->database->startTransaction('flood_unblock_clear');
diff --git a/modules/contrib/flood_unblock/src/FloodUnblockManagerInterface.php b/modules/contrib/flood_unblock/src/FloodUnblockManagerInterface.php
index e69de29..fed4334 100644
--- a/modules/contrib/flood_unblock/src/FloodUnblockManagerInterface.php
+++ b/modules/contrib/flood_unblock/src/FloodUnblockManagerInterface.php
@@ -0,0 +1,27 @@
+<?php
+
+namespace Drupal\flood_unblock;
+
+/**
+ * Interface for FloodUnblockManager.
+ */
+interface FloodUnblockManagerInterface {
+
+    /**
+     * Generate rows from the entries in the flood table.
+     *
+     * @return array
+     *   Entries of the flood table grouped by identifier (UID/IP).
+     */
+    public function getEntries();
+
+    /**
+     * The function that clear the flood.
+     *
+     * @param $event       The event variable.
+     * @param $identifier The identifier variable.
+     *
+     * @return string
+     */
+    public function flood_unblock_clear_event($event, $identifier);
+}
